build.gradle

```diff
@@ -24,4 +24,5 @@ dependencies {
     // https://mvnrepository.com/artifact/org.mockito/mockito-core
     testImplementation group: 'org.mockito', name: 'mockito-core', version: '2.23.4'
 
+    testImplementation 'io.cucumber:cucumber-java:6.8.1'
```

Comment:
A gradle most már tartalmazza a dependency-ket, így a lib mappa és tartalma elvileg felesleges, nem?
Ha van időd, akkor légyszi nézd meg, hogy nélküle is fut-e (lehet a gradle menüben reloadolni kell a projektet, hogy leszedje a függőségeket), és ha igen, akkor töröljük a felesleges jar file-okat.

Reply:
Igen, már azt használta, csak elfelejtettem törölni a lib mappát. Nem voltam a helyzet magaslatán tegnap, bocsi